ovirtbmc/ovirtbmc.py

```diff
@@ -43,29 +44,33 @@ def __init__(self, authdata, port, address, vm_name, cache_status, engine_fqdn,
         )
         self.vms_service = self.ovirt_conn.system_service().vms_service()
         self.vm = None
+        self.vm_name = None
         self.cache_disabled = not cache_status
         self.cached_status = None
         self.target_status = None
         # At times the bmc service is started before important things like networking have fully initialized. Keep
         # trying to find the vm indefinitely, since there's no point in continuing if we don't have an vm.
         while True:
             try:
-                self.vm = self._find_vm(vm_name)
-                if self.vm is not None:
-                    self.log(f'Managing vm: {vm_name} ID: {self.vm}')
-                    break
+                self.vm = self._find_vm(vm)
+                self.vm_name = self.vms_service.service(self.vm).get().name
+                self.log(f'Managing vm: {self.vm_name} ID: {self.vm}')
```

Comment:
This log message doesn't align with the convention used below where we print out `f'{self.vm_name} ({self.vm})'`. Let's keep things consistent

Reply:
Done